pom.xml

```diff
@@ -140,6 +140,14 @@
                 <configuration>
```

Comment:
[WARNING] Old version of checkstyle detected. Consider updating to >= v8.30
[WARNING] For more information see: https://maven.apache.org/plugins/maven-checkstyle-plugin/examples/upgrading-checkstyle.html

Reply:
Fixed